VideoLocker/res/layout/fragment_my_all_videos.xml

```diff
@@ -8,16 +9,16 @@
     
     <!-- keep this container, as playerfragment needs it to be present -->
     <LinearLayout
-        android:id="@+id/container_player"
+        android:id="@+id/my_all_videos_player_container"
```

Comment:
It's less confusing once we are using databinding. As 1zaman said the layout filename already includes "my_all_videos" so it's redundant to prefix the IDs with that.